src/views/TradingCompetition/helpers.ts

```diff
@@ -0,0 +1,8 @@
+export const localiseTradingVolume = (value: number, decimals = 0) => {
+  return value.toLocaleString('en-US', { maximumFractionDigits: decimals })
+}
+
+export const accountEllipsis = (account: string) =>
+  `${account.substring(0, 4)}...${account.substring(account.length - 4)}`
```

Comment:
We should make this a global helper.

Reply:
Nice, will follow up with a PR to do this 👌